src/services/PostService.ts

```diff
@@ -60,11 +60,11 @@ export class PostService {
   }
 
   
-  public async searchPosts(getSavedPostsRequest:getSavedPostsRequest): Promise<PostModel[]> {
+  public async searchPosts(getSearchedFeedbackRequest:GetSearchedFeedbackRequest): Promise<PostModel[]> {
```

Comment:
pls put space between parameter name and class name (i.e. `getSearchedFeedbackRequest: GetSearchedFeedbackRequest`)